### ISSUE TITLE
Check for multiple modules returns after first module without any message in version 2.0.1

### ISSUE THREAD

**Issue by norgeindian** [yireo/Yireo_ExtensionChecker#24]
I now included version `2.0.1`. We have a githook, which runs the check on all our custom modules and looks like that:
```
bin/magento module:status --enabled | grep -e CustomGento_  | while read -r MODULE; do echo "Checking $MODULE"; bin/magento yireo_extensionchecker:scan --module "$MODULE" --hide-needless 1 --hide-deprecated 1 || exit; done
```
As far as I see, this is exactly the way, it is advised in the README and this worked fine with version `1.2.17`.
But now, we only see the following output
`Checking CustomGento_FirstModule` and then the process fails without any further error message.
I tried it with the `-vvv` flag, but that did not change anything. Might it be, that we still need to adapt something in this command, to make it work in version 2?

**Comment by jissereitsma** [comment]
Another minor bug: The return codes in the `Scan` CLI were swapped: `0` means ok. I fixed this with 2.0.2.

**Comment by norgeindian** [comment]
Like always, thanks a lot for this quick fix. Works fine now.
